### ISSUE TITLE
Enterprise cloud setup wizard migration to the TEF

### ISSUE THREAD

**Issue by obriensystems** [terraform-google-modules/terraform-example-foundation#1135]
### TL;DR

Start procedures on removing the non-terraform parts of the ECS or a procedure to interleave the 2 
This particular case ended at a partial group creation where 2 of 10 groups were created on a rerun of step 2 but only made it to the gcp side not the admin side
<img width="1110" alt="Screenshot 2024-03-06 at 09 21 42" src="https://github.com/terraform-google-modules/terraform-example-foundation/assets/24765473/3bccb662-2fe3-4e76-bec5-409e12a13f1f">

using the groups left over from the cloud setup for now
https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/0-bootstrap/terraform.example.tfvars#L44
```
gcp-organization-admins
gcp-billing-admins
```

**Comment by fmichaelobrien** [comment]
stale bot timer restart - https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/.github/workflows/stale.yml#L21
